tests/ext/includes/skipif_requires_dev_env.inc

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
@SammyK what if name it `tests/ext/includes/skipif_no_dev_env.inc`

Reply:
Done. :)